dashboard/lib/contact_rollups_v2.rb

```diff
@@ -9,6 +9,9 @@ class ContactRollupsV2
     query_timeout: MAX_EXECUTION_TIME_SEC
   )
 
+  # Execute a SQL query in a transaction in the dashboard database.
+  # Does not return query results.
```

Comment:
Ok I think this seems clear now, but to confirm -- you would use `retrieve_query_results` if you need the results of a query, and `execute_query_in_transaction` if not?

Reply:
That's correct. I can add more explanation to both methods.